roles/roles_client.js

```diff
@@ -69,7 +65,7 @@ Object.assign(Roles, {
       unlessExists: false
     }, options)
 
-    const result = Meteor.roles.upsert({ _id: roleName }, { $setOnInsert: { children: [] } })
+    const result = RolesCollection.upsert({ _id: roleName }, { $setOnInsert: { children: [] } })
```

Comment:
This is breaking all our tests:

1. If we directly replace `Meteor.Roles` it will not apply this internal collection. We also can no longer check if a role is created using `Meteor.roles.countDocuments` 
2. If we use `hwillston:stub-collections` we can only test for integration and not manully stub specific methods for unit tests. This was possible when `Meteor.roles` was used.

Reply:
The `Meteor.roles` is still applied and unchanged, but you can now import he RolesCollection as well from the package.

Reply:
@jankapunkt please give this a look in the latest push, I have reverted to using the `Meteor` object for collection access.